sim/target-agnostic.mk

```diff
@@ -188,9 +186,11 @@ $(f1): $(header) $(DRIVER_CC) $(DRIVER_H) $(midas_cc) $(midas_h)
 	GEN_DIR=$(OUTPUT_DIR)/build OUT_DIR=$(OUTPUT_DIR) DRIVER="$(DRIVER_CC)" \
 	TOP_DIR=$(chipyard_dir)
 
-$(vitis): export CXXFLAGS := $(CXXFLAGS) $(common_cxx_flags) $(DRIVER_CXXOPTS)
+$(vitis): export CXXFLAGS := $(CXXFLAGS) $(common_cxx_flags) $(DRIVER_CXXOPTS) \
+	-std=c++14 -idirafter ${CONDA_PREFIX}/include -idirafter /usr/include -idirafter $(XILINX_XRT)/include
 # Statically link libfesvr to make it easier to distribute drivers to f1 instances
-$(vitis): export LDFLAGS := $(LDFLAGS) $(common_ld_flags) -Wl,-rpath='$$$$ORIGIN'
+$(vitis): export LDFLAGS := $(LDFLAGS) $(common_ld_flags) -Wl,-rpath='$$$$ORIGIN' \
+	-L${CONDA_PREFIX}/lib -Wl,-rpath-link=/usr/lib/x86_64-linux-gnu -L$(XILINX_XRT)/lib -luuid -lxrt_coreutil
```

Comment:
Just a note, all of the search-path arguments here can be removed once we switch to using the conda-packaged XRT libraries.  This was a temporary hack to make it so that things will compile & link correctly against the system-provided XRT on Ubuntu.  The paths need to be different on Centos.

Reply:
What's the state of the XRT conda package? Do we just need someone to actually do the switch over?

Reply:
We need to be able to bump to using 2022.1 and then start using it.  (Or, I could build older versions but I've been trying to avoid doing that).

I still don't see 2022.1 VM images in the Azure Portal.  @abejgonzalez mentioned trying 2022.1 on firesim1 or something at one point.

Reply:
I tried it at one point but until everyone can bump to 2022.1 successfully I held back from bumping everything (Bumping to 2022.1 breaks AWS builds supposedly @sagark )